tests/get_or_create/tests.py

```diff
@@ -484,3 +485,35 @@ def lock_wait():
         updated_person = Person.objects.get(first_name='John')
         self.assertGreater(after_update - before_start, timedelta(seconds=0.5))
         self.assertEqual(updated_person.last_name, 'NotLennon')
+
+
+class InvalidCreateArgumentsTests(TestCase):
+    msg = "Thing has no field named 'does_not_exist'"
+
+    def test_get_or_create_with_invalid_defaults(self):
+        with self.assertRaisesMessage(FieldDoesNotExist, self.msg):
+            Thing.objects.get_or_create(
+                name="a",
```

Comment:
use single quotes throughout params
also, if the params fit on the same line as `Thing.objects.get_or_create(` that's fine. You could change "does_not_exist" to "nonexistent"  and "some_value" to "b" to save a few characters if it helps with line length.